diff --git a/ios/SharingModule.swift b/ios/SharingModule.swift
index a918fa7592353662a9e42a8001f3b7302b36180f..d3b0e017784ef6de358a3c783dcc3465d3787c89 100644
--- a/ios/SharingModule.swift
+++ b/ios/SharingModule.swift
@@ -15,16 +15,8 @@ public final class SharingModule: Module {
       let activityController = UIActivityViewController(activityItems: [url], applicationActivities: nil)
       activityController.title = options.dialogTitle

-      activityController.completionWithItemsHandler = { type, completed, _, _ in
-        // user shared an item
-        if type != nil && completed {
-          promise.resolve(nil)
-        }
-
-        // dismissed without action
-        if type == nil && !completed {
-          promise.resolve(nil)
-        }
+      activityController.completionWithItemsHandler = { _, _, _, _ in
+        promise.resolve(nil)
       }

       guard let currentViewcontroller = appContext?.utilities?.currentViewController() else {
